src/scene.js

```diff
@@ -0,0 +1,50 @@
+import inquirer from 'inquirer';
+import { negativeFinish, positiveFinish } from './finishMessages.js';
+import { getPrompt } from './getPrompt.js';
+
+export class SceneInterface {
+
+    setStrategy(strategy) {
+    }
```

Comment:
to make it really abstract you'd better add throwing inside every interface function:
`throw new Error(...)`